src/GitLink/Linker.cs

```diff
@@ -131,5 +162,41 @@ public static bool Link(string pdbPath, LinkOptions options = default(LinkOption
 
             return true;
         }
+
+        private static void CreateSrcSrv(string srcsrvFile, SrcSrvContext srcSrvContext)
+        {
+            Argument.IsNotNull(() => srcSrvContext);
+            Argument.IsNotNullOrWhitespace(() => srcSrvContext.RawUrl);
+            Argument.IsNotNullOrWhitespace(() => srcSrvContext.Revision);
+            Argument.IsNotNullOrWhitespace(() => srcsrvFile);
+
+            if (srcSrvContext.VstsData.Count != 0)
+            {
+                File.WriteAllBytes(srcsrvFile, SrcSrv.CreateVsts(srcSrvContext.Revision, srcSrvContext.Paths, srcSrvContext.VstsData));
```

Comment:
We should add logging here so we can follow what decision making is done by the app.